Automattic/sensei
# Add Course Details step for Wizard.

Fixes #5128

### Changes proposed in this Pull Request
* Added styling for Course Details step.
* Title is being updated in real time into the post.
* Description is persisted (along with the title) in the wizard's data state as `courseDescription` and `courseTitle` respectively.

### Testing instructions
- Checkout to the branch.
- Go to create a new Course.
- Check:
  - That title is updated in the post.
  - That style looks correct.

### Screenshot / Video
<img width="1328" alt="image" src="https://user-images.githubusercontent.com/799065/170293936-580d712f-8c80-4c9a-8d4f-7117d0142f6e.png">
